elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/example/Country.java

```diff
@@ -9,18 +9,22 @@
 import com.yahoo.elide.datastores.aggregation.annotation.Cardinality;
 import com.yahoo.elide.datastores.aggregation.annotation.CardinalitySize;
 import com.yahoo.elide.datastores.aggregation.annotation.FriendlyName;
+import com.yahoo.elide.datastores.aggregation.queryengines.sql.annotation.FromTable;
 
 import lombok.Data;
 
 import javax.persistence.Entity;
 import javax.persistence.Id;
+import javax.persistence.Table;
 
 /**
  * A root level entity for testing AggregationDataStore.
  */
 @Data
 @Entity
 @Include(rootLevel = true)
+@FromTable(name = "countries")
```

Comment:
FromTable is only intended for AnalyticViews - not dimension tables.

Reply:
👍